### ISSUE TITLE
Filenames of defaults need to be changed to snake_case

### ISSUE THREAD

**Issue by robinvanemden** [MKaptein/streamingbandit#51]
Currently, the filenames of the default files are all camelCase - these need to be changed to snake_case:

```
 	getAction.py 	--> get_action.py 
	getContext.py 	--> get_context.py
	getReward.py 	--> get_reward.py
	setReward.py 	--> set_reward.py
```

**Comment by g0ulash** [comment]
Will be handled when defaults will get implemented. Working on it.

**Comment by g0ulash** [comment]
Fixed some defaults in c3055c2. Please try using AB_Test and Bootstrapped_Thompson_Sampling as defaults first. The rest will follow.
